<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewEmptyMapDialog</class>
 <widget class="QDialog" name="NewEmptyMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Map</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Title:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_titleEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="m_backgroundLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Background Color:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Permission :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_permissionSelector"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="m_formatGroup">
         <property name="title">
          <string>Format</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="m_landscapeCheckbox">
            <property name="text">
             <string>Landscape</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_portraitCheckbox">
            <property name="text">
             <string>Portrait</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_sqareCheckbox">
            <property name="text">
             <string>Square</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="m_sizeGroup">
       <property name="title">
        <string>Size (in pixels)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="m_smallRadio">
          <property name="text">
           <string>Small (%1 x %2 )</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_middleRadio">
          <property name="text">
           <string>Middle (%1 x %2 )</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_bigRadio">
          <property name="text">
           <string>Big (%1 x %2 )</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_hugeRadio">
          <property name="text">
           <string>Huge (%1 x %2 )</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_customRadio">
          <property name="text">
           <string>Custom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Custom</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Width:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="m_widthEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Height:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="m_heightEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewEmptyMapDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewEmptyMapDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_customRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_formatGroup</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_hugeRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_formatGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_bigRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_formatGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_middleRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_formatGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_smallRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_formatGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_customRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_hugeRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_bigRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_middleRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_smallRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
